lib/between_meals/knife.rb

```diff
@@ -80,6 +93,20 @@ def cookbook_upload(cookbooks)
       end
     end
 
+    def berks_cookbook_upload(cookbooks)
+      # cookbooks: array
+      # cookbook_paths: array
+      if cookbooks.any?
+        @cookbook_dirs.each do |path|
```

Comment:
no, it doesn't have that concept, it expects to find it's config file for a cookbook (`Berksfile`) in the CWD